brave-spancollector-scribe/src/test/java/com/github/kristofa/brave/scribe/ScribeReceiver.java

```diff
@@ -81,4 +77,11 @@ public ResultCode Log(final List<LogEntry> messages) throws TException {
         return result;
     }
 
+    public void setDelayMs(int delayMs) {
```

Comment:
I did :) but anyway I don't know how much attention this detail is worth.
Im fine either way